app/Models/TestDiff.php

```diff
@@ -0,0 +1,19 @@
+<?php
+
+declare(strict_types=1);
+
+namespace App\Models;
+
+use Illuminate\Database\Eloquent\Model;
+
+class TestDiff extends Model
+{
+    const TEST_TYPE_NOTRUN = 0;
+    const TEST_TYPE_FAILED = 1;
+    const TEST_TYPE_PASSED = 2;
+    const TEST_TYPE_FAILED_TIMESTATUS = 3;
```

Comment:
This might be a good place to use PHP 8's [Backed Enumerations](https://www.php.net/manual/en/language.enumerations.backed.php)